src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml

```diff
@@ -253,3 +253,9 @@ sylius_fixtures:
                                 name: 'Winter sale'
                                 channels:
                                     - 'FASHION_WEB'
+                                        
+                catalog_promotion_rule:
```

Comment:
There shouldn't be a possibility to define rules (and actions) independently of the catalog promotions. It should be implemented similarly to rules for cart promotion